plugins/tide-display/manager.py

```
@@ -499,20 +602,23 @@ def _moon_icon(self, draw, cx, cy, r, phase):
             draw.ellipse([dx, cy-r, dx+dark_w, cy+r], fill=C_BG)
         draw.ellipse(bbox, outline=_lerp(C_BG, C_MOON, 0.35), width=1)
 
-    def _txt(self, x, y, text, color=C_TEXT, small=True):
-        font = self.display_manager.extra_small_font if small else self.display_manager.small_font
-        try: self.display_manager.draw_text(text, x=x, y=y, font=font, color=color, centered=False)
+    def _raw_txt(self, x, y, text, color, font, centered):
+        try: self.display_manager.draw_text(text, x=x, y=y, font=font, color=color, centered=centered)
```

Ruff (0.16.0)

[error] Multiple statements on one line (colon)

(E701)